src/components/ContentsTable/index.jsx

```diff
@@ -2,8 +2,18 @@ import React from 'react'
 import style from './style.module.scss'
 import { CiEdit } from 'react-icons/ci'
 import { BiArchiveIn } from 'react-icons/bi'
+import PopUpMessage from '../../pages/Projects/PopUpDialog'
+import { Dialog } from 'primereact/dialog'
+import { Button } from 'primereact/button'
 
 const ContentsTable = (props) => {
+  const [visible, setVisible] = React.useState(false)
+  const [Edit, setEdit] = React.useState({})
+
+  const EditRow = (Edit) => {
```

Comment:
having `Edit` twice like this is quite confusing (first of all use camelCase, always camelCase unless it's a component or class name).
so maybe instead we can change to this ?
```suggestion
  const [edit, setEdit] = React.useState({})

  const EditRow = (rowData) => {
```